Source/FolioReaderPage.swift

```diff
@@ -425,7 +425,11 @@ public class FolioReaderPage: UICollectionViewCell, UIWebViewDelegate, UIGesture
     // MARK: UIMenu visibility
     
     override public func canPerformAction(action: Selector, withSender sender: AnyObject?) -> Bool {
-        if UIMenuController.sharedMenuController().menuItems?.count == 0 {
+		guard readerConfig.useReaderMenuController else {
+			return false
```

Comment:
Ok, this makes sense. As `return super.canPerformAction(action, withSender: sender)` is called anyway at the end of the method I have remove the guard lines.